### ISSUE TITLE
SAML configuration

### ISSUE THREAD

**Issue by m-rjoob** [grokability/snipe-it#15724]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

i configured SAML setup based on the documentation but when i try to login through SAML, the error in the attachment keep pop up
![Screenshot 2024-10-24 at 15 08 29](https://github.com/user-attachments/assets/9dab5c0e-8309-4353-b831-c234e2915759)

the integration is between the system and office 365

### Reproduction steps

i need help

### Expected behavior

i need additional support, any other reference to how to setup the SAML with office 365

### Screenshots

![Screenshot 2024-10-24 at 15 08 29](https://github.com/user-attachments/assets/aec26853-4617-4d30-b5ce-2998ea069799)

### Snipe-IT Version

Version v7.0.13 - build 15514 

### Operating System

Ubuntu

### Web Server

apache

### PHP Version

i don't know

**Comment by snipe** [comment]
Please provide your application logs so we can check whatever information might be logged there.

**Comment by m-rjoob** [comment]
> Please provide your application logs so we can check whatever information might be logged there.

could you please guide me on where I can export the logs

**Comment by m-rjoob** [comment]
can anyone help me here with my issue

**Comment by rderewianko** [comment]
Hi m-rjoob please read https://snipe-it.readme.io/docs/getting-help, step 3 shows logs.

<blockquote><img src="https://files.readme.io/e9d890e-small-standalone-transparent-400.png" width="48" align="right"><div>Snipe-IT Documentation</div><div><strong><a href="https://snipe-it.readme.io/docs/getting-help">Getting Help</a></strong></div><div>If you're having trouble setting up Snipe-IT, don't worry!</div></blockquote>

**Comment by m-rjoob** [comment]
@rderewianko  can you support me remotely to configure the SAML configuration, this session will be paid not free.
is this possible?

**Comment by rderewianko** [comment]
> @rderewianko can you support me remotely to configure the SAML configuration, this session will be paid not free. is this possible?

No, i'm not looking to be paid to help. If you're looking for [paid support](https://snipeitapp.com/support), i'd suggest you reach out to SnipeIT and they can assist. 

I'm assisting because I feel if I use a product the least I can do is give back to that project.

**Comment by m-rjoob** [comment]
That’s a great approach—supporting a project you use benefits both the project and its community.  

However, I really need help configuring SAML to allow my end users to log in through their Microsoft accounts to check the assets tagged to them. I’ve followed the documentation and set it up multiple times, but unfortunately, I’m still encountering the attached error.

**Comment by rderewianko** [comment]
Could you provide the application logs requested? Without that no one here is really going to be able to help you troubleshoot the issue.

Specifically, you may want to do the tail recommended in step 3, trigger the error and post those.

**Comment by m-rjoob** [comment]
@rderewianko  there is a pull request to fix this issue
one of my colleagues reviewed the code and fixed the issue and its worked, i logged in through SAML successfully.

https://github.com/snipe/snipe-it/pull/16020

**Comment by rderewianko** [comment]
So your PR request, from how I understand it you're not sending the correct value in nameID and instead you're changing the code? You're sending email (user.mail) but want to send sAMAccountName(user.userprincipalname)?

There seems to be more to this than you've explained.

But this is only guess based on your refusal to help us help you by providing any additional logs.
 
I don't work for SnipeIT, but after taking a brief look at the code you're changing—and without fully understanding the entire function—I would assume that this change could introduce bugs for others, since not everyone uses o365 and has saml properly working for their environment.

**Comment by m-rjoob** [comment]
<img width="1391" alt="Screenshot 2025-01-05 at 2 46 49 PM" src="https://github.com/user-attachments/assets/ca942e75-214e-49a7-af0c-303470e29202" />

Thank you for your response. I was trying to determine if there was anything I could do to assist, which is why I consulted my colleague, who happens to be a developer. I've attached a screenshot that shows the error displayed in the web browser when attempting to log in via SAML.

**Comment by snipe** [comment]
As we've requested multiple times, we need your application logs. https://snipe-it.readme.io/docs/getting-help#step-3-check-your-app-and-server-logs

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
